build.sh

```diff
@@ -27,15 +42,40 @@ create_dummy_ksp () {
     fi
     
     # TODO: Manual hack, a better way to handle this kind of identifiers may be needed.
-    if [ "$KSP_VERSION" == "0.90" ]
-    then
+    case $KSP_VERSION in
+    "0.90")
+        echo "Overiding '0.90' with '0.90.0'"
         KSP_VERSION="0.90.0"
-    fi
+        ;;
+    "1.0")
+        echo "Overiding '1.0.0' with '1.0.0'"
```

Comment:
Small detail, but I guess the first "1.0.0" should be "1.0" (And while we are at the small details, overiding -> overriding)? I'm not well enough versed in bash scripting to know how switch is implemented, but looks to me reasonable.